corehq/form_processor/models.py

```diff
@@ -321,7 +321,9 @@ def soft_delete(self):
     def to_json(self, include_attachments=False):
         from .serializers import XFormInstanceSQLSerializer
         serializer = XFormInstanceSQLSerializer(self, include_attachments=include_attachments)
-        return dict(serializer.data)
+        data = dict(serializer.data)
+        data['history'] = [dict(op) for op in data['history']]
```

Comment:
not thrilled with this - Django Rest Framework returns a `ReturnDict` object. I might rather update the `json_diff` function to be more clever or make a special case.